gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/SweRunner.java

```diff
@@ -15,22 +17,30 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
+import java.time.OffsetDateTime;
+
 /**
  * @author Theo Pascoli {@literal <theo.pascoli at rte-france.com>}
  */
 @Service
 public class SweRunner {
     private static final Logger LOGGER = LoggerFactory.getLogger(SweRunner.class);
     private final NetworkService networkImporter;
+    private final FileImporter fileImporter;
 
-    public SweRunner(NetworkService networkImporter) {
+    public SweRunner(NetworkService networkImporter, FileImporter fileImporter) {
         this.networkImporter = networkImporter;
+        this.fileImporter = fileImporter;
     }
 
     @Threadable
     public SweResponse run(SweRequest sweRequest) {
-        LOGGER.info("Request received for timestamp {}", sweRequest.getTargetProcessDateTime());
+        final OffsetDateTime targetProcessDateTime = sweRequest.getTargetProcessDateTime();
+        LOGGER.info("Request received for timestamp {}", targetProcessDateTime);
         Network network = networkImporter.importNetwork(sweRequest);
+        CimCrac cimCrac = fileImporter.importCimCrac(sweRequest.getCrac().getUrl());
+        Crac crac = fileImporter.importCrac(cimCrac, targetProcessDateTime, network);
```

Comment:
Can you make it in one line ?

Reply:
Added oneliner in Runner and added new method in FileImporter ;)